core/src/test/java/org/apache/iceberg/util/TestStructLikeMap.java

```diff
@@ -120,29 +120,28 @@ public void testRemove() {
 
     Map<StructLike, String> map = StructLikeMap.create(STRUCT_TYPE);
     map.put(record, "1-aaa");
-    Assert.assertEquals(1, map.size());
-    Assert.assertEquals("1-aaa", map.get(record));
-    Assert.assertEquals("1-aaa", map.remove(record));
-    Assert.assertEquals(0, map.size());
+    assertThat(map.size()).isOne();
+    assertThat(map).containsEntry(record, "1-aaa");
```

Comment:
`assertThat(map).hasSize(1).containsEntry(...)` and then you can remove the check above